src/screens/tabs/NFTs/screens/NftDetail/index.js

```diff
@@ -80,13 +82,18 @@ const NftDetail = ({ modalRef, handleClose, selectedNFT, ...props }) => {
         <Header
           center={
             <Text type="subtitle2">
-              {selectedNFT?.index ? `#${selectedNFT?.index}` : ''}
+              {isICNS
```

Comment:
can other nfts have the `name` prop? or only icns

Reply:
only icns